ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -5,7 +5,11 @@
 # shellcheck disable=SC1091
 . px4-alias.sh
 
-SCRIPT_DIR="$(CDPATH='' cd -- "$(dirname -- "$0")" && pwd)"
+#search path for sourcing rc.*
+PATH="$PATH:${R}etc/init.d-posix"
+
+#user defined variables for instances can be in PATH
+. rc.vars
```

Comment:
This is prone to errors: If a system has `rc.vars` in `PATH` already, the result is unexpected. To avoid this, prefix all with `px4-`, so this would be `px4-rc.vars`

Reply:
If so, naming convention for all rc.* files in init.d and init.d-posix directories should be revised.

Reply:
No, only for the ones that use relative paths, which you are changing now. It's currently all based on absolute paths.

Reply:
Ok